src/spectrecoff-cli/commands/Progress.fs

```diff
@@ -31,6 +32,12 @@ type Progress() =
             ChartItemWithColor ("Canvas", 0, Color.Red)
             ChartItemWithColor ("CanvasImage", 0, Color.Red)
             ChartItemWithColor ("TextPath", 0, Color.Red)
+            ChartItemWithColor ("TextPrompt", 0, Color.Red)
+            ChartItemWithColor ("Selection", 0, Color.Red)
+            ChartItemWithColor ("MultiSelection", 0, Color.Red)
```

Comment:
Not clear about these three. First of all, for me they are all part of the prompt module (just like chart), and then - what do you mean by selection? Isn't that what we have in the prompt module already?

Reply:
Yeah you are right, it was late in the train, not everyones best hours :( 
I just copied what I saw missing in the documentation, without thinking that it might be contained or planned in an existing module. 
Removed it, and adjusted the progress screenshot.